scopes/scope/export/export-cmd.ts

```diff
@@ -1,31 +1,27 @@
 import { Command, CommandOptions } from '@teambit/cli';
 import ejectTemplate from '@teambit/legacy/dist/cli/templates/eject-template';
-import { WILDCARD_HELP } from '@teambit/legacy/dist/constants';
+import { WILDCARD_HELP, COMPONENT_PATTERN_HELP } from '@teambit/legacy/dist/constants';
 import chalk from 'chalk';
 import { isEmpty } from 'lodash';
 import { ExportMain } from './export.main.runtime';
 
 export class ExportCmd implements Command {
-  name = 'export [component-patterns...]';
+  name = 'export';
   description = 'export components from the workspace to remote scopes';
   arguments = [
     {
       name: 'component-patterns...',
-      description:
-        'component IDs, component names, or component patterns (separated by space). Use patterns to export groups of components using a common scope or namespace. E.g., "utils/*" (wrap with double quotes)',
+      description: `(not recommended) ${COMPONENT_PATTERN_HELP}`,
     },
   ];
-  extendedDescription = `bit export => export all staged components to their current scope, if checked out to a lane, export the lane as well
-  \`bit export [id...]\` => export the given ids to their current scope
+  extendedDescription = `bit export => export all staged snaps/tags of components to their remote scope. if checked out to a lane, export the lane as well
+  \`bit export [pattern...]\` => export components included by the pattern to their remote scope (we recommend not using a pattern in
+    most scenarios so that all changes are exported simultaneously)
   ${WILDCARD_HELP('export remote-scope')}`;
```

Comment:
This actually should be only 'export', not 'export remote-scope'. (only legacy had the concept of exporting to specific scope from the export command)

Reply:
removed